index.html

```diff
@@ -289,6 +289,80 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <form class="contact-form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+
+        <input type="text" placeholder="Full name" />
+        <input type="text" placeholder="Email address"/>
+        <textarea placeholder="Enter text here"></textarea>
```

Comment:
Currently, a user can add any number of characters to the text field. Please consider adding a restriction of `max-length: 500` to restrict the number of characters. Moreover, there needs a restriction so a user can't leave the text area field empty. To achieve all this you can use the following piece of code as an example.

` <textarea placeholder="Enter text here" maxlength="500" required></textarea>`